src/inspector_js_api.cc

```diff
@@ -246,8 +246,11 @@ static void AsyncTaskScheduledWrapper(const FunctionCallbackInfo<Value>& args) {
 
   CHECK(args[0]->IsString());
   Local<String> task_name = args[0].As<String>();
-  String::Value task_name_value(args.GetIsolate(), task_name);
-  StringView task_name_view(*task_name_value, task_name_value.length());
+
+  std::vector<uint16_t> task_name_buffer(task_name->Length());
```

Comment:
It is better to use the `TwoByteValue` from `util.h`. It avoids memory allocations for strings smaller than 1024 characters. Currently it does not have a constructor for the `Local<String>`. It must be easy to add based on the existing constructor that takes the `Local<Value>`.

Reply:
It's a `Local<Value>` before it's a `Local<String>`, so a new constructor / conversation isn't even needed :smile: